tests/lib_files_test.py

```diff
@@ -111,11 +145,21 @@ def test_srw_delete(fc):
         "downloadFile",
         params=PKDict(
             simulation_type=fc.sr_sim_type,
-            simulation_id=d.models.simulation.simulationId,
             filename=u.basename,
         ),
+        data=PKDict(),
+        redirect=False,
     )
-    pkunit.pkre("not_used_name.zip.*does not exist", r.data)
+    pkunit.pkre("does not exist", pkcompat.from_bytes(r.data))
+
+    r = fc.sr_get_json(
+        "downloadLibFile",
+        params=PKDict(
+            simulation_type=fc.sr_sim_type,
+            filename=u.basename,
+        ),
+    )
+    pkunit.pkre("not_used_name.zip.*does not exist", pkcompat.from_bytes(r))
```

Comment:
I believe this should be an http status check for a 404 [see server_test.test_srw](https://github.com/radiasoft/sirepo/blob/3fa94dcbc7ec16897f12bdfb039eb3c0ce51b74a/tests/server_test.py#L94) for an example.

Reply:
We're getting 500 back here - is that a bug elsewhere?

Reply:
Possibly. The question is what do we want to return to the user: 

1) server error because we are generating the list and it should always be correct.
2) not found because that's meaningful even if the user can't correct it.

I suspect (1) now that I think about it. Where's it being generated (being lazy here, sorry :).

Reply:
`lib_file_abspath()` raises `sirepo.util.UserAlert`, which leads to the 500 status code

Reply:
That's a defect. UserAlert should get back to the user. I noticed lib_file_abspath also imports auth for no reason. What is upstream from the UserAlert? Create an issue for this with the stack and a better explanation. How is "not_used_name.zip.*does not exist" getting into the 500? Would be good to document that too.

Reply:
The UserAlert should not be a 500. It should be a 200.